examples/2016-10-31/custom_domains_with_route53_hosted_zone_name/template.yaml

```diff
@@ -0,0 +1,71 @@
+Parameters:
+  DomainName:
+    Type: String
+    Default: 'example.com'
+  ACMCertificateArn:
+    Type: String
+    Default: 'cert-arn-in-us-east-1'
+Resources:
+  MyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      InlineCode: |
+        exports.handler = async (event) => {
+          const response = {
+            statusCode: 200,
+            body: JSON.stringify('Hello from Lambda!'),
+          };
+          return response;
+        };
+      Handler: index.handler
+      Runtime: nodejs8.10
+      Events:
+        Fetch:
+          Type: Api
+          Properties:
+            RestApiId: !Ref MyApi
+            Method: Post
+            Path: /fetch
+
+  MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      OpenApiVersion: 3.0.1
+      StageName: Prod
+      Domain:
+        DomainName: !Ref DomainName
+        CertificateArn: !Ref ACMCertificateArn
+        EndpointConfiguration: EDGE
+        BasePath:
+          - /home
+        Route53:
+         HostedZoneName: www.my-domain.com.
+         IpV6: true
```

Comment:
I think `IpV6` is also optional

Reply:
This is an example, not saying these things are required, right?

Reply:
The line below says `Everything below here is optional` but `IpV6` is already optional so it should be under this comment

Reply:
Ahhh, I copied this from the `custom_domains_with_route53` example directory

Reply:
@timoschilling Ah, that comment was meant to point the difference between using your own distribution vs apigateway one. Not as "this property is optional".

Reply:
I actually don't see `IpV6` in the SAM spec, but from looking at the code behind this, it looks like it is optional

Reply:
Sorry didn't read the full comment, you both are right